src/lib/syncable_ledger/test_mask.ml

```diff
@@ -0,0 +1,108 @@
+open Core
```

Comment:
there's a file with the same name in `merkle_ledger_tests`, which could be confusing.